src/main/twirl/views/sectionPreview.scala.html

```diff
@@ -13,7 +13,7 @@
     f"RIFS$id%05d"
 }
 
-@main(s"Event title", backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
+@main(formSection.title, backLink=Some(BackLink("Return to application overview",controllers.routes.ApplicationController.show(appOverview.id).url))) {
```

Comment:
I've no opinion on that (well actually I think I would probably link it to the master page rather than the section ('RIFS - Application') but it's not a strong opinion!) but I've copied the current approach (?) so that's probably a question for Bola or possibly just general agreement/dictate ?  Trivial technical change either way but maybe something for a quick Monday morning discussion ?